src/components/Calculator.js

```diff
@@ -1,36 +1,67 @@
 import React, { Component } from 'react';
+import calculate from './logic/calculate';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: '',
+      next: '',
+      operation: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
   }
 
+  handleClick = (e) => {
+    const buttonName = e.target.name;
+    this.setState((prevState) => ({
+      ...calculate(prevState, buttonName),
+    }));
+  };
+
   render() {
+    const { total, next, operation } = this.state;
     return (
       <div className="container my-4">
         <div className="calculator card">
-          <input
-            type="text"
-            className="calculator-screen z-depth-1"
-            value=""
-            disabled
-          />
+          <div className="calculator-screen z-depth-1">
+            <span>{next || operation || total || ''}</span>
+          </div>
 
           <div className="calculator-keys">
-            <button type="button" className="operator btn btn-info" value="+">
+            <button
+              type="button"
+              className="operator btn btn-info"
+              name="+"
+              value="+"
+              onClick={this.handleClick}
+            >
```

Comment:
- [ ] As the previous reviewer mentioned You are missing the % button which should handle the modulus operation, and the +- button to convert numbers to either positive or negative. 
So Kindly add the modulus and +- operators and make sure that they do calculations as expected.